### DISCUSSION TITLE
Align content of HostedDropdown to the side

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#703] [Q&A] [answered]
Hi!

am I'm right, that it is not possible to align the dropdown to the left or the right side? There is this `TuiSideDirection`, but it is not possible to use it with `DropdownController`.

I have a collapsed side menu and I need to open a menu on click, but not at the bottom or the top. Do I miss something? Or are there other ways to accomplish this?

BTW, great library! I love it more every day!

Best regards,
Kersten

**Comment by waterplea** [comment, marked as answer]
There's `tuiDropdownSided` on `DropdownController`. It's undocumented for some reason, give it a shot, combined with `tuiDropdownAlign`

**Reply by loxy** [reply]
Cool. But `align` only respects a `TuiHorizontalDirection`. And even a `$any('bottom-left' | 'bottom-right' | 'top-left' | 'top-right')` does not change the setting. Is this the reason why it is undocumented?

**Reply by waterplea** [reply]
You use `tuiDropdownAlign` to set side, `tuiDropdownDirection` to set vertical direction and `[tuiDropdownSided]="true"`:
https://stackblitz.com/edit/taiga-sided-dropdown

**Reply by loxy** [reply]
Nice! I can prepare a PR to add this to the docs. Or should it stay undocumented for the moment?

**Reply by waterplea** [reply]
Sure, PR is welcome!
